cmake/GenConfig.cmake

```diff
@@ -0,0 +1,132 @@
+# --------------------------------------------------------------------------------------------
+#  Installation for CMake Module:  VISPConfig.cmake
+#  Part 1/3: ${BIN_DIR}/VISPConfig.cmake              -> For use *without* "make install"
+#  Part 2/3: ${BIN_DIR}/unix-install/VISPConfig.cmake -> For use with "make install"
+#  Part 3/3: ${BIN_DIR}/win-install/VISPConfig.cmake  -> For use within binary installers/packages
+# -------------------------------------------------------------------------------------------
+
+if(INSTALL_TO_MANGLED_PATHS)
+  set(VISP_USE_MANGLED_PATHS_CONFIGCMAKE TRUE)
+else()
+  set(VISP_USE_MANGLED_PATHS_CONFIGCMAKE FALSE)
+endif()
+
+if(HAVE_CUDA)
+  vp_cmake_configure("${CMAKE_CURRENT_LIST_DIR}/templates/VISPConfig-CUDA.cmake.in" CUDA_CONFIGCMAKE @ONLY)
+endif()
+
```

Comment:
ViSP doesn't use CUDA.
Should be removed as well as `templates/VISPConfig-CUDA.cmake.in`

Reply:
I don't see the interest of this new file. There is already `cmake/VISPGenerateConfig.cmake`